src/main/kotlin/uk/gov/gdx/datashare/repository/IngressEventType.kt

```diff
@@ -7,11 +7,11 @@ import org.springframework.data.domain.Persistable
 import org.springframework.data.relational.core.mapping.Column
 import java.time.LocalDateTime
 
-data class EventType(
+data class IngressEventType(
 
   @Id
   @Column("id")
-  val eventId: String,
+  val eventTypeId: String,
```

Comment:
Notice you are leaving this as is - iD not changing to UUID

Reply:
Yes as this is the full string of the event, like DEATH_NOTIFICATION